### PR TITLE
Add dedicated animation documentation

### PR DESCRIPTION
**Pull request by kathleenhang**
Closes [#1258](https://github.com/f3d-app/f3d/issues/1258)

I wrote a draft for the improved animation documentation which consists of:

- demonstration example
- command line options
- hotkey interactions
- supported file types
- time units

I used InterpolationTest.glb for the example since it was personally the most helpful animation file for me to better understand how animations work in F3D. Being new to 3D graphics, the idea of multiple animation sequences attached to one model was quite confusing for me at first.

This file is currently only available for F3D developers to download. I wonder if F3D can have an official download link for regular F3D users? That way, they can load up a good example of an animation-supported file and play around with it to better understand animation capability within F3D.

Currently, it seems most of the available example model files do not clearly demonstrate much of the animation aspect.

Feedback appreciated!

### CODE REVIEW

File: doc/user/ANIMATIONS.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## Demonstration
```
Comment (mwestphal) [inline doc/user/ANIMATIONS.md:1]: lets start with something more like:

F3D supports animaition, blablabla

File: doc/user/ANIMATIONS.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+This specific example uses an animation file which can be downloaded here: <\insert download link for: `InterpolationTest.glb`>\
```
Comment (mwestphal) [inline doc/user/ANIMATIONS.md:2]: you can link data from the repository directly

File: doc/user/ANIMATIONS.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+## Animation Interactions
+- Press <kbd>W</kbd> to cycle through animations
+- Press <kbd>Space</kbd> to play/pause animation
+## Supported File Types
+- ex2/.e/.exo/.g
+- .gltf/.glb
+- .fbx
+- .dae
+- .x
+- .usd
```
Comment (mwestphal) [inline doc/user/ANIMATIONS.md:39]: I dont think we should list format that supports animations.  Any format that supports animations, F3D should be able to show it.

File: doc/user/ANIMATIONS.md
Comment (mwestphal) [inline doc/user/ANIMATIONS.md:1]: Needs to be added to the doc itself.

**Comment by kathleenhang** [comment]
Okay fixed! How about the demonstration images? Are those good? I was planning to redo it with consistent image dimensions. The current one is a quick draft.

File: doc/user/ANIMATIONS.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+F3D can play animations for a number of file formats (.ex2/.e/.exo/.g, .gltf/.glb, .fbx, .dae, .x, .usd) if the file contains an animation.
```
Comment (mwestphal) [inline doc/user/ANIMATIONS.md:1]: How about the following ?

```
#Animations

If your file contains an animation, F3D is able to play it.
You can play it interactively or select a specific time to display.
If your files contains multiple animations you can show a single one or show all of them at the same time.
```

Feel free to improve :)

**Comment by mwestphal** [comment]
> Okay fixed! How about the demonstration images? Are those good? I was planning to redo it with consistent image dimensions. The current one is a quick draft.

I did not check yes, could you add clickable links in the description  :)

**Comment by kathleenhang** [comment]
> > Okay fixed! How about the demonstration images? Are those good? I was planning to redo it with consistent image dimensions. The current one is a quick draft.
> 
> I did not check yes, could you add clickable links in the description :)

Here is the link to the images:
https://github.com/f3d-app/f3d/blob/47b4d0629adc853e34071e469a95dd7cb0737a61/doc/user/ANIMATIONS.md

Are you able to see this? I am not sure if it is private. Here is another link that may work too:
https://raw.githubusercontent.com/f3d-app/f3d/47b4d0629adc853e34071e469a95dd7cb0737a61/doc/user/ANIMATIONS.md

**Comment by mwestphal** [comment]
> Are you able to see this? 

Indeed, it works!

And indeed, we should generate cleaner screenshots :)

**Comment by kathleenhang** [comment]
Thanks for the feedback! I just updated the screenshots and the introduction.

**Comment by mwestphal** [comment]
could you share these new screenshots ?

**Review by mwestphal — APPROVED** [review]

**Comment by mwestphal** [comment]
superseeded by https://github.com/f3d-app/f3d/pull/1481

**Pull request closed without merging**